authorization/src/main/java/com/terabite/authorization/service/SubscriptionService.java

```diff
@@ -0,0 +1,54 @@
+package com.terabite.authorization.service;
+
+import com.terabite.authorization.model.LoginStatus;
+import com.terabite.authorization.model.SubscribeRequest;
+import com.terabite.authorization.model.UserInformation;
+import com.terabite.authorization.repository.LoginRepository;
+import com.terabite.authorization.repository.UserRepository;
+import jakarta.transaction.Transactional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SubscriptionService {
+    private static final Logger log = LoggerFactory.getLogger(SubscriptionService.class);
+
+    private final LoginRepository loginRepository;
+
+    private final UserRepository userRepository;
+
+    public SubscriptionService(LoginRepository loginRepository, UserRepository userRepository) {
+        this.loginRepository = loginRepository;
+        this.userRepository = userRepository;
+    }
+
+    public ResponseEntity<?> subscribe(SubscribeRequest request) {
+        if (loginRepository.findByEmail(request.username()).isPresent()) {
+            log.info("Found email: {}", request.username());
+            UserInformation existingUser = userRepository.findByLogin_Email(request.username()).orElse(null);
+
+            if (existingUser != null && existingUser.getLogin() != null && existingUser.getLogin().getLoginStatus() == LoginStatus.LOGGED_IN) {
```

Comment:
Right here, you can remove this status check for LoginStatus. We will be doing authentication through JWT tokens. SO no need to have this. Also its causing an error when testing. Just remove this check and push and I will approve it.